src/http.rs

```diff
@@ -67,15 +73,31 @@ pub async fn routes() -> impl Filter<Extract = impl warp::Reply, Error = warp::R
     let all_docs_route = warp::path!(String / "_all_docs")
         .and(warp::get())
         .and(with_fdb(fdb.clone()))
-        .and(with_couch_directory(couch_directory))
+        .and(with_couch_directory(couch_directory.clone()))
```

Comment:
I think you can use a reference here instead of cloning:
`.and(with_couch_directory(&couch_directory))`